website/aggregated_questions.qmd

```diff
@@ -0,0 +1,251 @@
+# (Preliminary) Questions
+
+_The following section contains questions for consideration (pruning, subsectioning, and or rephrasing) in GET. Included are questions where some work has already been expended (thus, the author is less inclined to prune these questions), where little work has been expended (full critique welcome here), and questions that already exist on Metaculus. Ratings of importance for the questions is not included here yet but will be once they have been established. Information in brackets are the author's notes._
+
+## Ethical, Legal, & Societal Implications
+
+Will an international treaty [treaty required] regulating human genetic engineering be signed by 20XX? (note: conditional on legal genetic engineering)
+
+Will there cease to be a global moratorium on human germline genome editing by 20XX?
+
+Will a country ban polygenic screening for non-medical traits [exact traits required] by 20XX? (note: explicit ban, any country, specific trait)
+
+Will a significant [definition of significant required] legal case concerning discrimination [definition of discrimination required] based on genetic information [OR genetic engineering?] reach an international court by 20XX?
+
+Will over XX% of countries have laws in place that ban genetic modifications [scope of bans for current countries required] for non-health-related [better phrasing for "non-health-related" required] enhancements by 20XX?
+
+Will more than XX% of the world's population [OR consortium of countries OR specific country] oppose [definition of oppose required] the use of genetic editing for enhancement in public opinion polls [scope of polls required] by 20XX?
+
+Will any country legalize germline genome editing for enhancement by 20XX?
```

Comment:
modification: "loosen or repeal previous laws restricting genome editing for the purposes of enhancement"
since it has to be illegal to be legalized